### ISSUE TITLE
UniDict Issues

### ISSUE THREAD

**Issue by Virtuoel** [MinecraftModDevelopment/New-MMD-Public-Pack#10]
### Issues:
In the `UniDict.cfg` config file in the current pack version `B:keepOneEntry` is set to `true` as opposed to the default `false`. This removes OreDict tags from items that are owned by mods lower in the OreDict entry sorting order specified by the `S:ownerOfEveryThing` list, with those not included in it being the lowest priorities.  

### Examples:
This can cause issues for items that previously existed that weren't the higher in the sorting order, as they can no longer be converted to the items used in OreDict recipes, such as IC2 sulfur.  

Vanilla Iron Nuggets have also had `nuggetIron` removed from their OreDict entries, since ImmersiveEngineering adds an Iron Nugget and `minecraft` has been removed from the `S:ownerOfEveryThing` list, despite being one of the default values, which puts IE higher in the sorting order.  

### Potential Fixes:
A fix would be to set `B:keepOneEntry` to false, so conversion between OreDict entries can be done if desired, and to add `minecraft` back to the top of the `S:ownerOfEveryThing` list so Vanilla materials are not affected.  

I also suggest adding `mekanism` and `ic2` back to `S:ownerOfEveryThing` to more closely resemble the default values.

**Comment by Virtuoel** [comment]
The latest change mixes the priorities up again, putting Mekanism and IC2 above the priority of other mods like BaseMetals. The extra suggestion I meant was adding them back in in the same order as the defaults. Or is there some reason I haven't noticed behind putting Mek and IC2 at the top of the list?

Default:
```
S:ownerOfEveryThing <
    minecraft
    thermalfoundation
    substratum
    ic2
    mekanism
    immersiveengineering
    techreborn
>
```

c6ae59d:
```
S:ownerOfEveryThing <
    minecraft
    mekanism
    ic2
    basemetals
    modernmetals
    thermalfoundation
    thermalexpansion
    immersiveengineering
>
```

Suggestion:
```
S:ownerOfEveryThing <
    minecraft
    basemetals
    modernmetals
    thermalfoundation
    thermalexpansion
    ic2
    mekanism
    immersiveengineering
>
```

**Comment by ProxyNeko** [comment]
I will attempt this fix once I have fixed the server as that is stuck in a half updated stage due to an error but thanks for the pointer

**Comment by carstorm** [comment]
I did it the way I did due to not understanding what the fix was and following it step by step. Thank you for additional info.

**Comment by ProxyNeko** [comment]
This will be fixed in the next update I will make sure of it

**Comment by TheSpicePhantom** [comment]
@Virtuoel it says "default: false" but it's being generated with "true", maybe didnt see it, getting fixed

**Comment by ProxyNeko** [comment]
Hopefully fixed in 0.0.7 for good this time. Any more issues post back and I will open this up.. again..
